src/index.html

```diff
@@ -12,6 +12,148 @@
   </head>
   <body>
     <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="personal-info">
+        <legend>Personal information</legend>
+        <label for="surname">
+          Surname:
+          <input
+            type="text"
+            autocomplete="disable"
+            name="surname"
+            id="surname">
+        </label>
+        <label for="name">
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="disable"
```

Comment:
```suggestion
            autocomplete="off"
```
`off` а не `disable`